suc/pkg/service/service_rke2.go

```diff
@@ -0,0 +1,56 @@
+package service
+
+import (
+	"fmt"
+
+	"github.com/rancher/wins/pkg/defaults"
+	"github.com/sirupsen/logrus"
+)
+
+type Rke2Service struct {
+	Service
+}
+
+func OpenRke2Service() (*Rke2Service, bool, error) {
```

Comment:
```suggestion
func OpenRKE2Service() (*RKE2Service, bool, error) {
```
https://go.dev/wiki/CodeReviewComments#initialisms